src/App/Entity/AbstractDiscussion.php

```diff
@@ -179,6 +161,11 @@ public function getPosts()
         return $this->posts;
     }
 
+    public function getLastPost()
+    {
+        return $this->posts->last();
```

Comment:
last() is in reality using `end()` php function. It returns `false` if the array is empty.
I would prefer to return `null` if there is no posts. Like

```php
return $this->posts->isEmpty() ? null : $this->posts->last();
```

Reply:
Done